.github/workflows/static-analysis.yml

```diff
@@ -48,6 +48,7 @@ jobs:
              run: |
                  if [ ! -z "$PIMCORE_VERSION" ]; then
                        composer require --no-update pimcore/pimcore:"${PIMCORE_VERSION}"
+                       composer require --no-update pimcore/admin-ui-classic-bundle:1.x-dev
```

Comment:
@brusch Ok, i will stick with the `minimum-stability` `dev`  whenever possible.

> Mhh, i recall we had the stability dev and got removed at some point

Potentially, we encoutered the problem with [symfony/string 5.4.0-RC1](https://github.com/pimcore/web-to-print-bundle/actions/runs/10095467426/job/27915695285#step:9:325) in the past and that's why it got removed but by conflicting the string < 6.0 it works again.

It seems by looking at https://github.com/pimcore/web-to-print-bundle/pull/85 that it would also work, if we also add this exact line over there.
When not, as Blankse noted it would install the `admin-ui-classic-bundle` `1.5.0-RC2`.

I've merged https://github.com/pimcore/admin-ui-classic-bundle/pull/631 (which turns the 1.x-dev to upcoming `1.6` )and re-run by https://github.com/pimcore/web-to-print-bundle/actions/runs/10095554081/job/27954358251#step:6:178, but still opts for the RC2


Extra Note:
It is happening for all bundles that require admin ui bundle in the composer eg.
https://github.com/pimcore/portal-engine/actions/runs/9954046874/job/27498854015#step:11:220
https://github.com/pimcore/copilot-bundle/actions/runs/10091379995/job/27902826840#step:7:188
https://github.com/pimcore/data-hub/actions/runs/9804937618/job/27073621255#step:8:203

If confirmed, we would need a follow up in all the other bundles.